builder/builder-next/worker/worker_test.go

```diff
@@ -0,0 +1,79 @@
+package worker
+
+import (
+	"testing"
+
+	"github.com/containerd/platforms"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/stretchr/testify/require"
```

Comment:
Can you change this to use `gotest.tools` for asserting? We use testify only for a single package (functionality not provided by gotest.tools), but otherwise only use gotest.tools

Reply:
Thanks for the suggestion, done.